src/Controller/CustomerReorderAction.php

```diff
@@ -57,13 +63,18 @@ public function __construct(
         $this->reorderer = $reorderService;
         $this->urlGenerator = $urlGenerator;
         $this->session = $session;
+        $this->customer = $customer;
```

Comment:
Let's inject `CustomerContext` here, it'd make it possible to run on PHP-PM then.